teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
```

Comment:
- [ ] As the `Teacher` class inherits from `Person`, kindly make sure you update the constructor method to have the same behavior like the one of the `Person` class i.e. `name` should have default value of `Unknown`, `parent_permission` of true, and others. It should be something like this: 
```ruby
 ...
 def initialize(age, specialization, name = 'Unknown', parent_permission: true)
  ...
```